### ISSUE TITLE
Ready for external review: LUMIP experiment documentation

### ISSUE THREAD

**Issue by charliepascoe** [ES-DOC/esdoc-docs#165]
Review LUMIP experiments with MIP PIs.

**Comment by charliepascoe** [comment]
These experiments appear to be unused:
- deforest-reg-lnd
- deforest-reg-atm
- deforest-reg-gcm
- land-hist-1700

**Comment by charliepascoe** [comment]
These experiments need to be added:
- land-cCO2
- land-cClim

**Comment by charliepascoe** [comment]
KT's CMIP6_expt_list_042716 describes these experiments in the opposite sense to the CMIP6 overview of endorsed mips:
- land-noFire
- land-noWoodHarv
- land-noPasture
- land-crop-noIrrig
- land-crop-noFert

**Comment by charliepascoe** [comment]
The challenge with LUMIP is the blurring of whether a requirement is an instruction for a model configuration or is rather a kind of forcing.  Eg the treating of crop and pasture as unmanaged grassland as in land-crop-grass.

We will need the LUMIP team to review the land management forcings in particular.  I deduced the existance of land management forcings from the experiments that described their absence (required that they be set to 1850 conditions).

I identified the following land management forcings for LUMIP:
 * irrigation
 * fertilisation
 * wood harvest
 * fire management
